server/app/scanpy_engine/scanpy_engine.py

```diff
@@ -35,6 +35,10 @@ def __init__(self, data, args):
         self.diffexp_options = ["ttest"]
         self._create_schema()
 
+        # TODO: temporary work-arounds
+        self._IEEE754_X_warning_issued = False
```

Comment:
We don't seem to be using this variable, recommend deleting it

Reply:
oops - this is because I forgot to use it.   Great catch!